### PR TITLE
2014/07/26/rubykaigi-2014-registration-online (zh_cn)

### PR DESCRIPTION
**Pull request by dandananddada**
translate this post  2014/07/26/rubykaigi-2014-registration-online from english to chinese.

Related #1074

### CODE REVIEW

File: zh_cn/news/_posts/2014-07-26-rubykaigi-2014-registration-online.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+* 提交 1 次以上，75 折（13,500 日元）
+
+符合以上条件者请联系我们（2014 at rubykaigi dot org）。
+我们将寄给您与相应折扣等值的优惠券。
+
+## 非日本 Rubyist 请注意：
+
+Ruby Kaigi 2014，将给你带来：
+
+* 向Ruby 贡献者表示感谢的绝佳机会！！（RubyKaigi 会议将汇集来自世界各地的大量的Ruby贡献者！）
```
Comment (hlcfan) [inline zh_cn/news/_posts/2014-07-26-rubykaigi-2014-registration-online.md]: **大量的 Ruby 贡献者**

File: zh_cn/news/_posts/2014-07-26-rubykaigi-2014-registration-online.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+date: 2014-07-26 16:02:34 +0000
+lang: zh_cn
+---
+
+[RubyKaigi 2014](http://rubykaigi.org/2014) 报名页面现在已经上线。
+
+* 介绍：RubyKaigi 2014 是为 Rubyist 举办的为期三天双轨同期进行的技术交流会议
+* 地点： 日本东京
+* 时间： 九月十八日至九月二十日（星期三至星期五）
+* 人数： 超过550名rubyists
```
Comment (hlcfan) [inline zh_cn/news/_posts/2014-07-26-rubykaigi-2014-registration-online.md]: **超过550名 rubyists**

File: zh_cn/news/_posts/2014-07-26-rubykaigi-2014-registration-online.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+* [http://rubykaigi.org/2014](http://rubykaigi.org/2014)
+* [http://blog.rubykaigi.org](http://blog.rubykaigi.org)
+* [@rubykaigi](https://twitter.com/rubykaigi)
+* [http://lanyrd.com/2014/rubykaigi/](http://lanyrd.com/2014/rubykaigi)
+
+## 联系我们：
+
+2014 at rubykaigi dot org
+
+期待您的到来 <3
```
Comment (hlcfan) [inline zh_cn/news/_posts/2014-07-26-rubykaigi-2014-registration-online.md]: Im wondering do we need to translate such type of articles?

Comment (dandananddada) [inline zh_cn/news/_posts/2014-07-26-rubykaigi-2014-registration-online.md]: I think this article is no use.

**Comment by stomar** [comment]
Please amend the commits to use a commit message conforming to our conventions, like e.g. `Translate 2014/08/19/ruby-1-9-2-p330-released (zh_cn)`

**Comment by JuanitoFatas** [comment]
Thanks @dandananddada and @hlcfan, merged! :point_right: 75e9f5d

**Comment by stomar** [comment]
References to `zh_tw` fixed.

**Comment by JuanitoFatas** [comment]
> References to zh_tw fixed.

Thanks.

**Pull request closed without merging**